tests/integration/components/o-s-s/attribute-text-test.ts

```diff
@@ -0,0 +1,95 @@
+        await render(hbs`<OSS::Attribute::Text @label="Hello" />`);
+        assert.dom('.oss-attribute').exists();
+        assert.dom('.oss-attribute__value').hasText('-');
+      });
+
+      test('it renders with a tooltip when specified', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" @tooltip="Tooltip" />`);
+        assert.dom('.oss-attribute__info').exists();
+        await assert.tooltip('.oss-attribute__info').hasTitle('Tooltip');
+      });
+
+      test('it renders a dash as value when no value is specified', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" />`);
+        assert.dom('.oss-attribute').exists();
+        assert.dom('.oss-attribute__value').hasText('-');
+      });
+
+      test('it renders a dash as value when @value is empty', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" @value=""/>`);
+        assert.dom('.oss-attribute').exists();
+        assert.dom('.oss-attribute__value').hasText('-');
+      });
+    });
+
+    module('Copy action', function (hooks) {
+      hooks.beforeEach(function () {
+        this.textForCopy = 'copied value';
+        this.displayCopyBtn = true;
+      });
+
+      test('the text is copyable by default', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" @value={{this.textForCopy}} />`);
+
+        await triggerEvent('.oss-attribute', 'mouseenter');
+        assert.dom('.oss-attribute__copy').exists();
+      });
+
+      test('The text is copyable if the @copyable argument is truthy', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" @value={{this.textForCopy}} @copyable={{true}} />`);
+        await triggerEvent('.oss-attribute', 'mouseenter');
+        assert.dom('.oss-attribute__copy').exists();
+      });
+
+      test('The text is not copyable (no copy button) if the argument is falsy', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" @value={{this.textForCopy}} @copyable={{false}} />`);
+        await triggerEvent('.oss-attribute', 'mouseenter');
+        assert.dom('.oss-attribute__copy').doesNotExist();
+      });
+
+      test("The text is not copyable if text value isn't specified", async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" @copyable={{true}} />`);
+        await triggerEvent('.oss-attribute', 'mouseenter');
+        assert.dom('.oss-copy--inline').doesNotExist();
+      });
+
+      test('The text is not copyable if text value is specified but the value is empty', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" @value="" @copyable={{true}} />`);
+        await triggerEvent('.oss-attribute', 'mouseenter');
+        assert.dom('.oss-copy--inline').doesNotExist();
+      });
+
+      test('The copy action is successfully called', async function (assert) {
```

Comment:
This is already tested at the `OSS::Copy` component level; no need to duplicate here :)